src/components/m-footer-languages/_template.js

```diff
@@ -8,7 +8,7 @@ export default function ({ title, items, short }) {
     <ul class="m-footer-languages__list">
       ${Array.isArray(items) && items.map(({ name, code = 'en', url = '', isActive }) => html`
         <li class="m-footer-languages__list-item">
-          <a class="${classnames('m-footer-languages__link', {
+          <a class="${classnames('m-footer-languages__link', 'js-footer-languages__link', {
             'is-footer-languages-active': isActive,
           })}" href="${url}" lang="${code}" data-language="${code}">${short ? code : name}</a>
```

Comment:
Same here `data-language` can be removed.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/lang